src/appshell/qml/MainToolBar.qml

```diff
@@ -32,6 +32,8 @@ Item {
     height: radioButtonList.height
 
     property alias navigation: navPanel
+    property alias navigation: navCtrl
```

Comment:
this line is added to avoid merge conflict, sorry for forgetting pull before my work